packages/terra-site/src/examples/form/examples/TextareaField.jsx

```diff
@@ -1,50 +1,96 @@
 import React from 'react';
-// eslint-disable-next-line import/no-extraneous-dependencies
 import TextareaField from 'terra-form/lib/TextareaField';
 
-const TextareaFieldExamples = () => (
-  <form>
-    <TextareaField
-      inputId="name"
-      label="Name"
-      name="name"
-      defaultValue="Mike"
-      error="Name is required"
-      help="The name given to you at birth."
-    />
-    <TextareaField
-      inputId="profile-description"
-      label="Profile Description"
-      name="profile_description"
-      error="Name is required"
-      help="The name given to you at birth."
-      cols={10}
-      maxLength={15}
-      minLength={5}
-      rows={10}
-      required
-    />
-    <div>
-      <TextareaField
-        inputId="difficult-problems"
-        label="Tell us about your most difficult programming problem"
-        name="difficult_problem"
-        isInline
-        error="This field is required"
-      />
-      <TextareaField
-        label="How well do you work in a team?"
-        name="teamwork"
-        inputId="teamwork"
-        isInline
-        error="This field is required"
-        help="The name given to you at birth."
-        maxLength={15}
-        minLength={5}
-        required
-      />
-    </div>
-  </form>
-);
+class TextareaFieldExamples extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: 'Mike',
+      description: 'Michael',
+      problem: 'Chicken McNuggets',
+      teamwork: 'Tron',
+    };
+    this.handleNameChange = this.handleNameChange.bind(this);
+    this.handleDescriptionChange = this.handleDescriptionChange.bind(this);
+    this.handleProblemsChange = this.handleProblemsChange.bind(this);
+    this.handleTeamworkChange = this.handleTeamworkChange.bind(this);
+  }
 
+  handleNameChange(event) {
+    this.setState({ name: event.target.value });
+  }
+
+  handleDescriptionChange(event) {
+    this.setState({ description: event.target.value });
+  }
+
+  handleProblemsChange(event) {
+    this.setState({ problem: event.target.value });
+  }
+
+  handleTeamworkChange(event) {
+    this.setState({ teamwork: event.target.value });
+  }
+
+  render() {
+    return (
+      <div>
+        <form>
+          <TextareaField
+            inputId="name"
+            label="Name"
+            name="name"
+            defaultValue="Mike"
+            error="Name is required"
+            help="The name given to you at birth."
+            onChange={this.handleNameChange}
+          />
+          <TextareaField
+            inputId="profile-description"
+            label="Profile Description"
+            name="profile_description"
+            error="Profile Descirption is required"
+            help="How would you describe your current position?"
+            cols={10}
+            maxLength={15}
+            minLength={5}
```

Comment:
This maxLength caught me off guard on the examples site, the TextArea occupies a large section of the screen but is restricted to 15 characters